contracts/kleros/KlerosGovernor.sol

```diff
@@ -66,6 +68,7 @@ contract KlerosGovernor is Arbitrable {
 
     uint public submissionDeposit; // Value in wei that needs to be paid in order to submit the list. Note that this value should be higher than arbitration cost.
```

Comment:
We can remove `Note that this value should be higher than arbitration cost.` now. Do the same for `changeSubmissionDeposit`